jbpm-designer-backend/src/test/java/org/jbpm/designer/web/server/CalledElementServletTest.java

```diff
@@ -84,4 +93,51 @@ public void testDoPostProfileAlreadySet() throws Exception {
                never()).findProfile(any(HttpServletRequest.class),
                                     anyString());
     }
+
+    @Test
+    public void testGetJavaTypeNames() throws Exception {
+        final PackageDataModelOracle oracle = mock(PackageDataModelOracle.class);
+        when(oracle.getProjectPackageNames()).thenReturn(Arrays.asList("org"));
+
+        ModelField[] modelFields = new ModelField[]{
+                new ModelField("this",
+                               "org.Address",
+                               ModelField.FIELD_CLASS_TYPE.REGULAR_CLASS,
+                               ModelField.FIELD_ORIGIN.SELF,
+                               FieldAccessorsAndMutators.ACCESSOR,
+                               DataType.TYPE_THIS),
+                new ModelField("street",
+                               String.class.getName(),
+                               ModelField.FIELD_CLASS_TYPE.REGULAR_CLASS,
+                               ModelField.FIELD_ORIGIN.DECLARED,
+                               FieldAccessorsAndMutators.BOTH,
+                               DataType.TYPE_STRING),
+                new ModelField("homeAddress",
+                               Boolean.class.getName(),
+                               ModelField.FIELD_CLASS_TYPE.REGULAR_CLASS,
+                               ModelField.FIELD_ORIGIN.DECLARED,
+                               FieldAccessorsAndMutators.BOTH,
+                               DataType.TYPE_BOOLEAN),
+                new ModelField("number",
+                               Integer.class.getName(),
+                               ModelField.FIELD_CLASS_TYPE.REGULAR_CLASS,
+                               ModelField.FIELD_ORIGIN.DECLARED,
+                               FieldAccessorsAndMutators.BOTH,
+                               DataType.TYPE_NUMERIC_INTEGER)};
+
+        Map<String, ModelField[]> fields = new java.util.HashMap<>();
+        fields.put("org.Address",
+                   modelFields);
+
+        when(oracle.getProjectModelFields()).thenReturn(fields);
+
+        CalledElementServlet servlet = new CalledElementServlet();
+        List<String> javatypeNameList = servlet.getJavaTypeNames(oracle);
+
+        assertNotNull(javatypeNameList);
+        assertEquals(1,
+                     javatypeNameList.size());
+        assertEquals("org.Address",
+                     javatypeNameList.get(0));
+    }
```

Comment:
AssertJ will let you replace these asserts with `Assertions.assertThat(javaTypeNameList).containsOnly("org.Address")`.

Reply:
Thanks Triceo - there is very similar test code in kie-wb as well so if its ok with you ill create a cleanup Jira to add your suggestion in both places. Let me know if thats ok to do in a separate commit?

Reply:
Sure, no problem.